### ISSUE TITLE
Documentation of extra fields {pak} adds to the `DESCRIPTION`

### ISSUE THREAD

**Issue by wurli** [r-lib/pak#598]
From the pak documentation, I've found some discussion of a [few fields](https://pak.r-lib.org/reference/pkg_deps.html?q=DESCRIPTION#value) which are added to the `DESCRIPTION` files of packages installed using pak. However, as a package developer, I would find it helpful to have a single page to document these and explain how they are used by pak, and how they are intended to be used by other parties. 

Many thanks - pak is an amazing package 😃

**Comment by gaborcsardi** [comment]
Thanks! They are not intended to be used by other parties currently, hence they are also not documented.

**Comment by wurli** [comment]
Fair enough! Perhaps there's then a question about whether this should be more explicitly noted in the documentation. AFAIK, `DESCRIPTION` files _are_ intended to be read by users (although doubtless this is happens fairly rarely), and documentation definitely [does exist](https://cran.r-project.org/doc/manuals/R-exts.html#The-DESCRIPTION-file) for the standard fields. Perhaps simply a list of the fields which pak adds would be helpful, so users can at least get a quick idea of which fields are non-standard?
